DirectionsPlayground/DirectionsPlayground/Sources/QueriesList.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import SwiftUI
+
+struct QueriesList: View {
+    @State
+    var queries: [Query] = [] {
```

Comment:
Should we add the first Query to the screen or should we add text explaining that you need to add a query to see how Directions work? It might be confusing to see the empty list